packages/plugins/src/hardhat/tasks.ts

```diff
@@ -222,12 +403,14 @@ task(TASK_CHUGSPLASH_COMMIT)
 
 task(TASK_CHUGSPLASH_VERIFY)
   .setDescription('Checks if a deployment config matches a bundle hash')
-  .addParam('configUri', 'location of the config file')
+  // TODO: Should `deployConfig` be changed to `configUri` so that deployConfig
+  // has a consistent meaning across tasks?
+  .addParam('deployConfig', 'location of the config file')
```

Comment:
Yeah `deployConfig` should always be a path to the config file, while `configUri` is the remote config URI